src/video/windows/SDL_windowsevents.c

```diff
@@ -1267,7 +1267,14 @@ WIN_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
 
         /* We'll do our own drawing, prevent flicker */
     case WM_ERASEBKGND:
+        if (!data->videodata->cleared)
```

Comment:
What is the interaction with an application that starts with a white loading screen?

Reply:
Maybe I should make it a hint then. Every game I've ever done starts with black, but I can see the rationale.

But the other annoying thing here is that Windows has been wildly inconsistent about what it paints the window with. Even if you set a specific brush for it to use on the first draw in the CreateWindow(Ex?) function, Windows seems to ignore that value these days, and will just fill it white unless you do something special in `WM_ERASEBKGND`.

Reply:
For what it's worth, SDL intentionally unconditionally clears the window contents to black on macOS when the window is created: https://github.com/libsdl-org/SDL/blob/main/src/video/cocoa/SDL_cocoawindow.m#L1557

Reply:
You're right. I checked a game that I've seen with a white loading screen, and the initial window is black before switching to a white loading screen.